src/pulse3D/plate_recording.py

```diff
@@ -373,6 +421,31 @@ def __next__(self):
             raise StopIteration
 
 
+def load_files(path):
+    with tempfile.TemporaryDirectory() as recording_dir:
+        # get file names
+        if path.endswith('.zip'):
+            zf = zipfile.ZipFile(path)
+            zf.extractall(path=recording_dir)
+        else:
+            recording_dir = path
+        # sort H5 files
+        h5_files = glob.glob(os.path.join(recording_dir, "**", "*.h5"), recursive=True)
+        recording_files = [f for f in h5_files if "Calibration" not in f]
+        calibration_files = [f for f in h5_files if "Calibration" in f]
+        # create WellFiles
+        tissue_well_files = [None] * len(recording_files)
+        baseline_well_files = [None] * len(calibration_files)
+        for file_list, well_file_list in (
+            (recording_files, tissue_well_files),
+            (calibration_files, baseline_well_files)
+        ):
+            for f in file_list:
+                well_file = WellFile(os.path.join(recording_dir, f))
```

Comment:
I don't think this needs the `os.path.join` since `f` already includes the `recording_dir` part in it